packages/router/src/adapters/subgraph/subgraph.ts

```diff
@@ -488,46 +488,48 @@ export const getTransactionForChain = async (
   if (!sdk) {
     throw new ContractReaderNotAvailableForChain(chainId, { method, methodId });
   }
```

Comment:
If the problem is the sdk is not configured (see #735 ) will the added try-catch fix it?